fetch.bs

```diff
@@ -1113,8 +1113,8 @@ a <a>byte-case-insensitive</a> match for one of
  request object is copied, but will be removed if the request is modified by unprivileged APIs.
 
  <p>`<code>Range</code>` headers are commonly used by <a lt="download the hyperlink">downloads</a>
- and <a lt="resource fetch algorithm">media fetches</a>, although neither of these currently specify
- how. <a href=https://github.com/whatwg/html/pull/2814>html/2914</a> aims to solve this.
+ and <a lt="resource fetch algorithm">media fetches</a>. More details can be found in
+ <a href=https://github.com/whatwg/html/pull/7655>html/7655</a>.
```

Comment:
```suggestion
 and <a lt="resource fetch algorithm">media fetches</a>.
```
Since the PR landed I don't think we should reference it directly anymore.